tox.ini

```diff
@@ -4,7 +4,7 @@
 [tox]
 skipsdist=True
 skip_missing_interpreters = True
-envlist = lint, static-{charm, lib}, unit
+envlist = lint, static-{charm, lib}, unit, scenario
 
 [vars]
 src_path = {toxinidir}/src/
```

Comment:
By convention these vars do not have a trailing `/` and when used below we do use a `/`.